src/tokens/tokens.service.ts

```diff
@@ -401,6 +401,8 @@ class TokenListener implements EventListener {
 
     const commonData = {
       id: transferId,
+      location: 'address=' + event.address,
```

Comment:
Make this via URLSearchParams:
```
new URLSearchParams({
      address: event.address
})
```

Reply:
If there is more than one will these be comma separated? Or `&` separated like query parameters? They should be comma separated. Also, is the order predictable?

Reply:
Ah, didn't know the need to be comma separated. This should be fine then